README.md

```diff
@@ -28,16 +28,18 @@ ViteHelper\Vite\ViteHelper:
   manifestDir: '/public/manifest.json'
 ```
 
-E.g., if you use TypeScript, change the `mainJs` prop to `"main.ts"`. If you set `forceProductionMode` to true, only build files will be served - which will happen on your live site anyway. 
-Set the `devHostNeedle` to distinguish your live site from your local environment. 
+*ViteHelper Config setting options:*
+- If you set `forceProductionMode` to true, only build files will be served when running `npm run build` - which will happen on your live site anyway.
+  - Setting this property to false will provide live updates when running `npm run dev` on a local environment.
+- Set the `devHostNeedle` to distinguish your live site from your local environments domain extension.
+- Set the `devPort` to an unused port (not the same port where the site is hosted from) for the Vite server to build asset from.
```

Comment:
This is a bit misleading. The dev port is not for build assets.

Reply:
"This port will be used by the Vite server" is not correct because vite does not know about the yaml config.

See the current readme, I did some changes to clarify. Tldr: The devPort must be configured in both configs, or you let vite decide and take that port (usually 3000 if available).